public/components/common/flyout.tsx

```diff
@@ -51,62 +52,206 @@ export const Flyout = () => {
     >
       <EuiFlyoutHeader hasBorder>
         <EuiTitle size="m">
-          <h2 id="flyoutTitle">Help</h2>
+          <h2 id="flyoutTitle">
+            <FormattedMessage id="searchRelevance.flyout.title" defaultMessage="Help" />
+          </h2>
         </EuiTitle>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
         <EuiText>
-          <h3>Query format</h3>
+          <h3>
+            <FormattedMessage
+              id="searchRelevance.flyout.sectionHeading"
+              defaultMessage="Query format"
+            />
+          </h3>
           <p>
-            Enter a query in{' '}
+            <FormattedMessage
+              id="searchRelevance.flyout.sectionDetails"
+              defaultMessage="Enter a query in"
+            />{' '}
             <EuiLink
               target="_blank"
               href="https://opensearch.org/docs/latest/opensearch/query-dsl/index/"
             >
-              OpenSearch Query DSL
+              <FormattedMessage
+                id="searchRelevance.flyout.sectionDetailsLink"
+                defaultMessage="OpenSearch Query DSL"
+              />
             </EuiLink>
-            . Use the <EuiCode>%SearchText%</EuiCode> variable to refer to the text in the search
-            bar. When you enter <strong>Search</strong>, the queries are sent to the search engine
-            using the <EuiCode>GET</EuiCode> HTTP method and the <EuiCode>_search</EuiCode>{' '}
-            endpoint.
+            <FormattedMessage
```

Comment:
Is it possible to wrap this entire in a single `FormattedMessage` block? May need to use brackets `{}` for the sub-components for formatting, like `strong`.

Reply:
Thanks for the suggestion! I made some changes to consolidate some of the `FormattedMessage` blocks.